src/components/ToggleTestModeButton.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useCallback, useRef, useState } from 'react'
+import Button from './Button'
+import Modal from './Modal'
+import Prose from './Prose'
+
+export interface Props {
+  isTestMode: boolean
+  togglingTestMode: boolean
```

Comment:
This will require a search-and-replace, but…
```suggestion
  isTogglingTestMode: boolean
```

Reply:
Done.